src/billable-services/cash-point/add-cash-point.modal.tsx

```diff
@@ -104,7 +104,10 @@ const AddCashPointModal: React.FC<AddCashPointModalProps> = ({ closeModal, onCas
 
   return (
     <>
-      <ModalHeader closeModal={closeModal} title={t('addCashPoint', 'Add Cash Point')} />
+      <ModalHeader
+        closeModal={closeModal}
+        title={cashPointToEdit ? t('editCashPoint', 'Edit Cash Point') : t('addCashPoint', 'Add Cash Point')}
```

Comment:
```suggestion
        title={cashPointToEdit ? t('editCashPoint', 'Edit cash point') : t('addCashPoint', 'Add cash point')}
```

Sentence case is more canonical for translation strings.